app/controllers/claims_controller.rb

```diff
@@ -106,6 +106,10 @@ def claim_params
     )
   end
 
+  def claim_submitted
```

Comment:
Maybe this could be named after what it does rather than when it happens? e.g. `mail_claim_submitted` or something.